source/includes/api/_microservices.md

```diff
@@ -0,0 +1,129 @@
+# Microservices
+
+A list of information related to the microservices
```

Comment:
Please add a warning saying these endpoints are only available to users with role ADMIN